lib/galaxy/jobs/output_checker.py

```diff
@@ -130,12 +133,15 @@ def check_output(tool, stdout, stderr, tool_exit_code, job):
             # log.debug( "Tool did not define exit code or stdio handling; "
             #          + "checking stderr for success" )
             if stderr:
-                peak = stderr[0:250]
-                log.debug("Tool produced standard error failing job - [%s]" % peak)
                 state = DETECTED_JOB_STATE.GENERIC_ERROR
             else:
                 state = DETECTED_JOB_STATE.OK
 
+        if DETECTED_JOB_STATE != DETECTED_JOB_STATE.OK and stderr:
+            if stderr:
```

Comment:
Is this additional if needed?

Reply:
Depends what stderr is. I suppose it might be an empty string, in which case we'd produce a meaningless log message. If it's actually None it'd fail at `stderr[0:ERROR_PEAK]`